Detectors/TRD/calibration/macros/checkNoisyMCMs.C

```diff
@@ -51,13 +67,15 @@ void checkNoisyMCMs(std::string inpFile = "trdtracklets.root", bool enableWriteC
   tree->SetBranchAddress("TrackTrg", &trigInPtr);
   tree->SetBranchAddress("Tracklet", &trackletsInPtr);
 
-  std::array<std::pair<unsigned int, unsigned short>, constants::MAXHALFCHAMBER * constants::NMCMHCMAX> trackletCounter{};
+  std::array<std::pair<unsigned int, unsigned int>, constants::MAXHALFCHAMBER * constants::NMCMHCMAX> trackletCounter{};
```

Comment:
Actual bug was here, since unsigned short is not sufficient for an index up to 69120..